realsense2_camera/launch/rs_launch.py

```diff
@@ -108,7 +109,7 @@ def launch_setup(context, params, param_name_suffix=''):
         return [
             launch_ros.actions.Node(
                 package='realsense2_camera',
-                namespace=LaunchConfiguration('camera_name' + param_name_suffix),
+                namespace=(LaunchConfiguration("camera_namespace"), "/", LaunchConfiguration('camera_name' + param_name_suffix)),
```

Comment:
as I mentioned above, 
I think this line should be:
namespace=LaunchConfiguration('camera_namespace' + param_name_suffix),

Reply:
namespace also needs suffix??
I think namespace should be kept what user passed.


if that is applied. topic would be like this.
namespace1/cam1
namespace2/cam2

Reply:
With this code,
```python
launch_ros.actions.Node(
                package='realsense2_camera',
                namespace=LaunchConfiguration("camera_namespace"),
                name=LaunchConfiguration('camera_name' + param_name_suffix),
                executable='realsense2_camera_node',
                parameters=[params
                            , params_from_file
                            ],
                output='screen',
                arguments=['--ros-args', '--log-level', LaunchConfiguration('log_level' + param_name_suffix)],
                emulate_tty=True,
                )
```

publish the topics like this.

```
/test/aligned_depth_to_color/camera_info
/test/aligned_depth_to_color/image_raw
/test/aligned_depth_to_color/image_raw/compressed
/test/aligned_depth_to_color/image_raw/compressedDepth
/test/aligned_depth_to_color/image_raw/theora
/test/color/camera_info
/test/color/image_raw
/test/color/image_raw/compressed
/test/color/image_raw/compressedDepth
/test/color/image_raw/theora
/test/color/metadata
/test/depth/camera_info
/test/depth/image_rect_raw
/test/depth/image_rect_raw/compressed
/test/depth/image_rect_raw/compressedDepth
/test/depth/image_rect_raw/theora
/test/depth/metadata
/test/extrinsics/depth_to_color
/test/extrinsics/depth_to_depth
/test/imu
```

Reply:
But,

With this code,
```python
            launch_ros.actions.Node(
                package='realsense2_camera',
                namespace=(LaunchConfiguration("camera_namespace"), "/", LaunchConfiguration('camera_name' + param_name_suffix)),
                name=LaunchConfiguration('camera_name' + param_name_suffix),
                executable='realsense2_camera_node',
                parameters=[params
                            , params_from_file
                            ],
                output='screen',
                arguments=['--ros-args', '--log-level', LaunchConfiguration('log_level' + param_name_suffix)],
                emulate_tty=True,
                )
```

the result is like this.
```
/test/camera/aligned_depth_to_color/camera_info
/test/camera/aligned_depth_to_color/image_raw
/test/camera/aligned_depth_to_color/image_raw/compressed
/test/camera/aligned_depth_to_color/image_raw/compressedDepth
/test/camera/aligned_depth_to_color/image_raw/theora
/test/camera/color/camera_info
/test/camera/color/image_raw
/test/camera/color/image_raw/compressed
/test/camera/color/image_raw/compressedDepth
/test/camera/color/image_raw/theora
/test/camera/color/metadata
/test/camera/depth/camera_info
/test/camera/depth/image_rect_raw
/test/camera/depth/image_rect_raw/compressed
/test/camera/depth/image_rect_raw/compressedDepth
/test/camera/depth/image_rect_raw/theora
/test/camera/depth/metadata
/test/camera/extrinsics/depth_to_color
/test/camera/extrinsics/depth_to_depth
/test/camera/imu
```